### DISCUSSION TITLE
tsai-Rocket

### DISCUSSION THREAD

**Discussion by ayan1c2** [timeseriesAI/tsai#243] [Q&A] [answered]
I am using spyder5 IDE with python 3.6.6, TF 2.6 under Anaconda dist.

I did - pip install tsai to install tsai and all models are running except the ROCKET

Code as taken from github -

bs = 16
c_in = 7  # aka channels, features, variables, dimensions
c_out = 2
seq_len = 15
xb = torch.randn(bs, c_in, seq_len).to(default_device())

m = ROCKET(c_in, seq_len, n_kernels=1_000, kss=[7, 9, 11]) # 1_000 for testing with a cpu. Default is 10k with a gpu!
test_eq(m(xb).shape, [bs, 2_000])

Exception coming at - m(xb) 
Response:

torch.Size([16, 7, 15])
Traceback (most recent call last):

  File "C:\Users\AyanC\git_ai_projects\SiameseNet\Tsai_anomalous_data_forecast_daily.py", line 192, in <module>
    test_eq(m(xb).shape, [bs, 2_000])

  File "c:\users\ayanc\appdata\local\programs\python\python36\lib\site-packages\torch\nn\modules\module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)

  File "c:\users\ayanc\appdata\local\programs\python\python36\lib\site-packages\tsai\models\ROCKET.py", line 124, in forward
    for i in progress_bar(range(self.n_kernels), display=self.verbose, leave=False, comment='kernel/kernels'):

TypeError: __init__() got an unexpected keyword argument 'comment'

Can you please help me out?

**Comment by vrodriguezf** [comment]
What version of `fastprogress` do you have?

**Reply by hellozhq** [reply]
I also encounter the problem。fastprogress is 1.0.2

**Comment by oguiza** [comment]
What version of tsai are you using? 
Have you installed it and restarted the kernel?
I'm not sure what the issue is. It works fine for me.

**Comment by z-979** [comment]
Have you solved the problem now？

**Comment by z-979** [comment, marked as answer]
I have solved this problem. I have reinstalled a new environment. The setup is as follows. It can run.
Maybe it's the problem of **cpu** and **gpu**
```
os              : Windows-10-10.0.19044-SP0
python          : 3.8.13
tsai            : 0.3.4
fastai          : 2.7.10
fastcore        : 1.5.27
fastprogress    : 1.0.3
torch           : 1.8.1
device          : 1 gpu (['NVIDIA GeForce MX450'])
cpu cores       : 4
threads per cpu : 2
RAM             : 15.79 GB
GPU memory      : [2.0] GB
```
